app/controllers/organizations/adopter_fosterer/liked_pets_controller.rb

```diff
@@ -0,0 +1,59 @@
+class Organizations::AdopterFosterer::LikedPetsController < Organizations::BaseController
+  before_action :authenticate_user!
+  before_action :set_pet, only: [:create]
+  before_action :set_liked_pet, only: [:destroy]
+  layout "adopter_foster_dashboard", only: :index
+
+  def index
```

Comment:
My intention was to get this up and running then we can decide what to do here and make a future issue for it.

Reply:
I think it's nicer for the user to see a pet they have liked has been adopted, rather than just have it disappear from their liked pets, which could be a bit confusing. Therefore, I propose we add the badge if a pet has been adopted. To that end, let's wait until your PR for fosters has been merged, because in order to show the badge we need to determine if the pet has been adopted, and that required the changes to the match, i.e., from singular to plural, on the Pet model.

Reply:
I agree that it would probably be confusing for adopters if liked pets just disappeared. Sounds good!